### PR TITLE
[office-js, office-js-preview] Removing HTML tables from requirement set and host descriptions

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by AlexJerabek** [comment]
Please do not merge without another maintainer's approval.

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * **{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}**: Restricted
```

Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: I would prefer all of these to exclude the colon from being bolded

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Hmm, this prop is weird. By keeping just the Outlook info, it seems like it's only supported by Outlook and not the other hosts.

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
             * This property is introduced in Outlook in [requirement set](/office/dev/add-ins/reference/requirement-sets/outlook-api-requirement-sets) Mailbox 1.3 but without the `options` parameter. Support for `options` is added in Mailbox Preview.
```

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: If you accept the suggestion for the previous comment, then remove this sentence.

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
        * **Hosts**: Word, Excel, Outlook, PowerPoint
```

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Shouldn't this one include pointing folks to the section on the CustomXmlParts reqt. set?

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Keep with links to appropriate sections on SOT?

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Does this need all the coercion items?

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Should include Requirement sets "header"

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * **Requirement sets** 
```

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * **Requirement sets**: [CustomXmlParts](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets#customxmlparts)
```

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: missing Requirement sets "header"

**Review by ElizabethSamuel-MSFT — CHANGES_REQUESTED** [review]
These are my comments so far.

File: types/office-js-preview/index.d.ts
Comment (AlexJerabek) [reply types/office-js-preview/index.d.ts]: Follow up: is the Source property only an Outlook thing? Or was that table inaccurate?

File: types/office-js-preview/index.d.ts
Comment (AlexJerabek) [reply types/office-js-preview/index.d.ts]: They are called out in the SoT conceptual page for this method.  Again, this is just my logic. I'm open to a better system.

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: I can't say if source is Outlook only. Looks that way.

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: Where else can a user get this info?

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: ok

**Comment by AlexJerabek** [comment]
Thanks for the review @ElizabethSamuel-MSFT. A new version has been committed.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
@AlexJerabek ready to merge?

**Comment by AlexJerabek** [comment]
@uniqueiniquity Go to go. Thanks!

**Pull request merged**
